e2e/tests/manageDocuments_test.js

```diff
@@ -0,0 +1,227 @@
+  I.seeInTab(['Correspondence uploaded by HMCTS 2', 'Document name'], 'Email with evidence attached');
+  I.seeInTab(['Correspondence uploaded by HMCTS 2', 'Notes'], 'Case evidence included');
+  I.seeInTab(['Correspondence uploaded by HMCTS 2', 'Date and time uploaded'], dateFormat(submittedAt, 'd mmm yyyy'));
+  I.seeInTab(['Correspondence uploaded by HMCTS 2', 'File'], 'mockFile.txt');
+  I.seeTextInTab(['Correspondence uploaded by HMCTS 2', 'Uploaded by']);
+
+  I.seeInTab(['Correspondence uploaded by local authority 1', 'Document name'], 'Correspondence document');
+  I.seeInTab(['Correspondence uploaded by local authority 1', 'Notes'], 'Test notes');
+  I.seeInTab(['Correspondence uploaded by local authority 1', 'Date and time uploaded'], dateFormat(submittedAt, 'd mmm yyyy'));
+  I.seeInTab(['Correspondence uploaded by local authority 1', 'File'], 'mockFile.txt');
+  I.seeTextInTab(['Correspondence uploaded by local authority 1', 'Uploaded by']);
+  I.seeTextInTab(['Correspondence uploaded by local authority 1', ''], 'Confidential');
+  I.seeInTab(['Correspondence uploaded by local authority 2', 'Document name'], 'C2 supporting document');
+  I.seeInTab(['Correspondence uploaded by local authority 2', 'Notes'], 'Supports the C2 application');
+  I.seeInTab(['Correspondence uploaded by local authority 2', 'Date and time uploaded'], dateFormat(submittedAt, 'd mmm yyyy'));
+  I.seeInTab(['Correspondence uploaded by local authority 2', 'File'], 'mockFile.txt');
+  I.seeTextInTab(['Correspondence uploaded by local authority 2', 'Uploaded by']);
+});
+
+Scenario('HMCTS Admin and LA upload confidential C2 supporting documents', async ({I, caseViewPage, manageDocumentsEventPage, manageDocumentsLAEventPage, uploadC2DocumentsEventPage}) => {
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+  await manageDocumentsForLAHelper.uploadC2(I, caseViewPage, uploadC2DocumentsEventPage);
+  await caseViewPage.goToNewActions(config.applicationActions.manageDocumentsLA);
+
+  manageDocumentsEventPage.selectC2SupportingDocuments();
+  await manageDocumentsEventPage.selectC2FromDropdown();
+  await I.goToNextPage();
+  await manageDocumentsEventPage.uploadConfidentialSupportingEvidenceDocument(supportingEvidenceDocuments[0]);
+  await I.addAnotherElementToCollection();
+  await manageDocumentsEventPage.uploadSupportingEvidenceDocument(supportingEvidenceDocuments[1]);
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.applicationActions.manageDocumentsLA);
+
+  caseViewPage.selectTab(caseViewPage.tabs.c2);
+
+  I.seeInTab(['C2 Application 1', 'C2 supporting documents 1', 'Document name'], 'Email to say evidence will be late');
+  I.seeInTab(['C2 Application 1', 'C2 supporting documents 1', 'Notes'], 'Evidence will be late');
+  I.seeInTab(['C2 Application 1', 'C2 supporting documents 1', 'File'], 'mockFile.txt');
+  I.seeTextInTab(['C2 Application 1', 'C2 supporting documents 1', ''], 'Confidential');
```

Comment:
is 'Uploaded by' and 'Date and time uploaded' also shown for C2 supporting? should be I think since it is SupportingEvidenceBundle - if its there lets check it with I.seeTextInTab